PolyEngine/Engine/Src/Movement/MovementSystem.cpp

```diff
@@ -15,6 +15,9 @@ void MovementSystem::MovementUpdatePhase(Scene* world)
 {
 	float deltaTime = (float)(TimeSystem::GetTimerDeltaTime(world, Poly::eEngineTimer::GAMEPLAY));
 	InputWorldComponent* inputCmp = world->GetWorldComponent<InputWorldComponent>();
+	
+	if (inputCmp->GetIsConsumed())
```

Comment:
This is very hacky

Reply:
I know. Please see reported issues with camera rotation and mouse dragging on Imgui panels.